### PR TITLE
List only active backfills by default

### PR DESCRIPTION
**Pull request by bergman**

### CODE REVIEW

File: styx-cli/src/main/java/com/spotify/styx/cli/Main.java [hunk 1]
```diff
@@ -562,7 +571,7 @@ public Subparser parser(Subparsers subCommands) {
   }
 
   private enum BackfillCommand {
-    LIST("ls", "List backfills"),
+    LIST("ls", "List backfills, by default, only active backfills are printed. Use option -a (--show-all) to show all"),
```
Comment (honnix) [inline styx-cli/src/main/java/com/spotify/styx/cli/Main.java]: A bit too long shown in the terminal. How about `List active backfills. Use option -a (--show-all) to show all`? Or we just rely user to do `styx backfill list -h` and just say `List active backfills`

File: styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java [hunk 2]
```diff
@@ -548,8 +548,75 @@ void deleteResource(String id) throws IOException {
     return Optional.of(entityToBackfill(entity));
   }
 
-  List<Backfill> getBackfills() {
-    final EntityQuery query = Query.entityQueryBuilder().kind(KIND_BACKFILL).build();
+  List<Backfill> getBackfills(boolean showAll) {
```
Comment (honnix) [inline styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java]: These methods could likely be refactored by defining a private method taking in `queryBuilder` and doing the actual query; the other methods just call it.

File: styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java [hunk 3]
```diff
@@ -548,8 +548,75 @@ void deleteResource(String id) throws IOException {
...
+    final EntityQuery.Builder queryBuilder = Query.entityQueryBuilder().kind(KIND_BACKFILL);
+    if (!showAll) {
+      queryBuilder
```
Comment (honnix) [inline styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java]: could be extracted as a method

File: styx-cli/src/main/java/com/spotify/styx/cli/Main.java (hunk 1 above)
Comment (bergman) [inline styx-cli/src/main/java/com/spotify/styx/cli/Main.java]: Agreed, taking your suggested text :)

**Comment by bergman** [comment]
@honnix I made the changes you requested, it's much better now so thanks :)

**Comment by honnix** [comment]
LGTM 👍

**Pull request merged**